### PR TITLE
Create zh_CN translation

### PR DESCRIPTION
**Pull request by woodjecket**
Create zh_CN translation.

### CODE REVIEW

File: assets/bundles/bundle_zh_CN.properties [hunk 1]
```diff
@@ -0,0 +1,747 @@
...
+block.betamindy-block-unpacker.name = [scarlet]方块解包器[]
+从更大的方块卸载流体和方块。\n[scarlet]已弃用，请用新版本代替[]
+block.betamindy-block-factory.name = [scarlet]方块工厂[]
+block.betamindy-block-factory.description = 将输入的结构升级到6x6大小，需要冷却。 \n[scarlet]已弃用，请用新版本代替[]
+
+# Environment
+block.betamindy-radiation.name = 辐射
+block.betamindy-exotic-matter.name = 异域物品
+block.betamindy-present.name = 礼物
+block.betamindy-present.description = A holly jolly block. Click to unwrap, you don't know what's inside!冬青枝装饰的好玩方块，点击以打开，你不会知道里面是什么的。
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: Remove English

File: assets/bundles/bundle_zh_CN.properties [hunk 2]
```diff
@@ -0,0 +1,747 @@
...
+block.betamindy-item-shop.name = 物品商店
+block.betamindy-item-shop.description = 用猫猫币购买物品\n[accent]用贸易驿站获取猫猫币，并用猫猫银行分配。[]
+block.betamindy-unit-shop.name = 单位商店
+block.betamindy-unit-shop.description = 用猫猫币购买单位\n[accent]用贸易驿站获取猫猫币，并用猫猫银行分配。[]
+block.betamindy-extra-shop.name = 额外商店
+block.betamindy-extra-shop.description = 用猫猫币购买物品包和补贴。\n用猫猫币购买物品。\n[accent]用贸易驿站获取猫猫币，并用猫猫银行分配。[]
+block.betamindy-trading-post.name = 贸易驿站
+block.betamindy-trading-post.description = 售卖物品以获取猫猫币\n[accent]连接到猫猫币节点以分配[]
+block.betamindy-coin-source.name = 猫猫币源
+block.betamindy-coin-source.description = Maintains the Anucoin balance, removing or creating endless Anucoins when connected to an Anubank. Sandbox only.\n[accent]Get Anucoins with a Trading Post and distribute them with an Anucoin Node.[]维持猫猫币平衡，连接到猫猫银行时创造或删除无休止的猫猫币。\n[accent]用贸易驿站获取猫猫币，并用猫猫币节点分配。[]
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: Remove English

File: assets/bundles/bundle_zh_CN.properties [hunk 3]
```diff
@@ -0,0 +1,747 @@
...
+block.betamindy-anucoin-node.name = \uE005猫猫银行
+block.betamindy-anucoin-node.description = 管理猫猫币和经济的中心责任建筑。能连接到产出或消费猫猫币的建筑。连接到猫猫仓库来提升上限。
+block.betamindy-anucoin-safe.name = \uE005猫猫保险箱
+block.betamindy-anucoin-safe.description = 增加连接银行的猫猫币的上限，在断开连接后猫猫币会回到银行。
+block.betamindy-anucoin-vault.name = \uE005猫猫仓库
+block.betamindy-anucoin-vault.description = 更大的猫猫保险箱，增加连接银行的猫猫币的上限，在断开连接后猫猫币会回到银行。
+
+block.betamindy-cafe.name = 小餐馆
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: 咖啡馆

File: assets/bundles/bundle_zh_CN.properties [hunk 4]
```diff
@@ -0,0 +1,747 @@
...
+block.betamindy-gamer-wood.name = [pink]木块地板[]
+
+# Campaign
+block.betamindy-altar.name = 古代祭坛
+block.betamindy-terraformer.name = 地球拯救者
+block.betamindy-terraformer-c.name = 地球拯救者 C
+
+# Items
+item.betamindy-bittrium.name = 字节元素
+item.betamindy-bittrium.description = 存在于 [#ef8aff]Portal Invasions[]。用作货币或在古代发明中使用。
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: Portal Invasions -> 传送门入侵

File: assets/bundles/bundle_zh_CN.properties [hunk 5]
```diff
@@ -0,0 +1,747 @@
...
+block.betamindy-buttonpad-large.description =  踩上时发电。也能侦测上方的空中单位。\n涵盖多个格子。
+block.betamindy-buttonrouter.name = 按钮路由器
+block.betamindy-buttonrouter.description = 输入物品时发电。行为和正常路由器一样。
+block.betamindy-buttonrouter-large.name = 按钮分配器
+block.betamindy-buttonrouter-large.description = 输入物品时发电。 行为和正常分配器一样……对吧
+block.betamindy-isotope-reactor.name = 同位素反应堆
+block.betamindy-isotope-reactor.description = 使用异域放射性矿物发电。
+
+# Units
+block.betamindy-repair-turret.name = 再生塔
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties:228]: 修复塔？

File: assets/bundles/bundle_zh_CN.properties [hunk 6]
```diff
@@ -0,0 +1,747 @@
...
+
+# UI
+ui.teamalert = 队伍 {0} 遭到队伍 {1} 攻击！
+ui.teamalertsilent = 队伍 {0} 遭到攻击！
+ui.notagain = 不再显示
+ui.bloomplease = 建议打开[accent]{0}[]设置以获取最佳游戏体验。是否打开？
+ui.musicmodplease = 为了获取最佳游戏体验，可以下载BetaMindy音乐包： [accent]{0}[]是否下载？
+
+ui.hardmode.title = Portal Invasion
+ui.hardmode.intro = A [#ef8aff]Portal Invasion[] 开始了！
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: Portal Invasion -> 传送门入侵

File: assets/bundles/bundle_zh_CN.properties [hunk 7]
```diff
@@ -0,0 +1,747 @@
...
+ui.hardmode.hudIntro = [accent]<[] 什么 [accent]>[]
+ui.hardmode.level = [accent]（等级 {0}）[]
+ui.hardmode.lv = [accent]Lv{0}[]
+ui.hardmode.win = [accent]清除侵略！[]
+ui.hardmode.lose = [accent]游戏结束！[]
+ui.hardmode.demoend = Portal Invasion的演示版到此结束！更多内容尽请期待！
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: Portal Invasion -> 传送门入侵

File: assets/bundles/bundle_zh_CN.properties [hunk 8]
```diff
@@ -0,0 +1,747 @@
...
+setting.ifritview.name = 自由长度
+
+# UI
+ui.teamalert = 队伍 {0} 遭到队伍 {1} 攻击！
+ui.teamalertsilent = 队伍 {0} 遭到攻击！
+ui.notagain = 不再显示
+ui.bloomplease = 建议打开[accent]{0}[]设置以获取最佳游戏体验。是否打开？
+ui.musicmodplease = 为了获取最佳游戏体验，可以下载BetaMindy音乐包： [accent]{0}[]是否下载？
+
+ui.hardmode.title = Portal Invasion
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: 传送门入侵

File: assets/bundles/bundle_zh_CN.properties [hunk 9]
```diff
@@ -0,0 +1,747 @@
...
+ui.ground = 陆军
+ui.naval = 海军
+ui.cart = 运送
+ui.sell = 卖出
+ui.sellAccept = 你确定要卖出
+ui.sellAccept2 = 为了
+ui.transaction = 交易
+ui.trans.button = 总计： \uE005{0} 需要\uE005[scarlet]{1}[]
+ui.trans.linked = [coral]已连接[]
+ui.trans.error = [scarlet]出现了一些错误。[] 请尝 [accent]刷新[]此页面。
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties:662]: `[scarlet]出现了一些错误。[]请尝式[accent]刷新[]此页面。`

File: assets/bundles/bundle_zh_CN.properties [hunk 10]
```diff
@@ -0,0 +1,747 @@
...
+purchase.nucleus-in-a-box.description = 一个终代核心
+purchase.repair-kit.name = 修理包
+purchase.repair-kit.description = 2 个修复器， 2 修复点
+purchase.spore-kit.name = 孢子包
+purchase.spore-kit.description = 滑稽的数字
+
+purchase.first-aid.name = 急救
+purchase.first-aid.description = 把所有核心治疗到最高生命值
+purchase.invi-core.name = 无敌核心
+purchase.invi-core.description =队伍所有核心无敌十秒钟\n\n然后治疗到满血
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties:708]: Add space after =

File: assets/bundles/bundle_zh_CN.properties [hunk 11]
```diff
@@ -0,0 +1,747 @@
...
+
+# Overrides
+lenum.shoot = 向某个位置射击。\n也可控制[sky]风扇[]、[sky]聚光灯[]和[sky]抓手[]。
+laccess.color = 照明器发光的颜色\m也可以控制音符盒：\n[#ffaaaa]乐器[] [#aaffaa]音高[[0, 84)[] [#aaaaff]音量 [[0, 10.0][]
+status.boss.name = Boss
+
+# TODO Units: Newton, Joule, Watt, Kelvin, Becquerel
+
+# Hints
+hint.breakCrystal = 引爆[accent]爆炸采掘器[] 以收获水晶，拆除得不到任何东西
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties:745]: Remove space

File: assets/bundles/bundle_zh_CN.properties [hunk 12]
```diff
@@ -0,0 +1,747 @@
...
+# block.block-forge.description = Forges block payloads. Use with the payload cannon to remotely place down blocks.
+# block.block-forge.details = The forbidden block, revealed by BetaMindy(tm).
+block.betamindy-block-workshop.name = [scarlet]错误的构造器[]
+block.betamindy-block-workshop.description = 锻造大型方块，需要水来处理方块。\n[scarlet]已弃用，请用新版本代替[]
+block.betamindy-payload-destroyer.name = [scarlet]错误的解构器[]
+block.betamindy-payload-destroyer.description =  Anuke你到底要对我做甚么？\n[scarlet]已弃用，请用新版本代替[]
+block.betamindy-block-packer.name = [scarlet]方块打包器[]
+block.betamindy-block-packer.description = 将方块装载到更大的方块上。\n[scarlet]已弃用，请用新版本代替[]
+block.betamindy-block-unpacker.name = [scarlet]方块解包器[]
+从更大的方块卸载流体和方块。\n[scarlet]已弃用，请用新版本代替[]
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties:86]: `block.betamindy-block-unpacker.description = `

File: assets/bundles/bundle_zh_CN.properties [hunk 13]
```diff
@@ -0,0 +1,747 @@
...
+block.betamindy-spinner.details = 一通电就开始转动并拾起方块。切断电源则停止转动，并在无遮挡的地方放下当前方块。
+block.betamindy-spinner-infi.name = 无限轴承
+block.betamindy-spinner-infi.description = 能转动最多120个前部周围的方块，此轴承本身作为旋转轴。单击以改变旋转方向。能把方块拾起为载荷，附带孢子史莱姆才可以粘贴多个方块。仅限沙盒。
+block.betamindy-spinner-infi.details = 为什么是120？大概是因为超过这个数就要溢出了。
+
+block.betamindy-spinner-inert.name = 惯性轴承
+block.betamindy-spinner-inert.description = 能转动最多16个前部周围的方块，这种旋转轴承即使被其他齿轮拾取为载荷也不停止转动。
+block.betamindy-spinner-inert-infi.name = 无限惯性轴承
+block.betamindy-spinner-inert-infi.description = 能转动最多120个前部周围的方块，此轴承本身作为旋转轴。这种旋转轴承即使被其他齿轮拾取为载荷也不停止转动。仅限沙盒。
+block.betamindy-spinner-inert-infi.details =是无限惯性轴承，的确，我真不擅长起名。
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: Add space after =

File: assets/bundles/bundle_zh_CN.properties [hunk 14]
```diff
@@ -0,0 +1,747 @@
...
+从更大的方块卸载流体和方块。\n[scarlet]已弃用，请用新版本代替[]
+block.betamindy-block-factory.name = [scarlet]方块工厂[]
+block.betamindy-block-factory.description = 将输入的结构升级到6x6大小，需要冷却。 \n[scarlet]已弃用，请用新版本代替[]
+
+# Environment
+block.betamindy-radiation.name = 辐射
+block.betamindy-exotic-matter.name = 异域物品
+block.betamindy-present.name = 礼物
+block.betamindy-present.description = A holly jolly block. Click to unwrap, you don't know what's inside!冬青枝装饰的好玩方块，点击以打开，你不会知道里面是什么的。
+block.betamindy-present.details = [scarlet]圣诞节[] [lime]快乐[] [scarlet]暨[] [lime]新[] [scarlet]年[] [lime]快乐[][scarlet]![]
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: Remove spaces

File: assets/bundles/bundle_zh_CN.properties [hunk 15]
```diff
@@ -0,0 +1,747 @@
...
+
+block.betamindy-spinner-inert.name = 惯性轴承
+block.betamindy-spinner-inert.description = 能转动最多16个前部周围的方块，这种旋转轴承即使被其他齿轮拾取为载荷也不停止转动。
+block.betamindy-spinner-inert-infi.name = 无限惯性轴承
+block.betamindy-spinner-inert-infi.description = 能转动最多120个前部周围的方块，此轴承本身作为旋转轴。这种旋转轴承即使被其他齿轮拾取为载荷也不停止转动。仅限沙盒。
+block.betamindy-spinner-inert-infi.details =是无限惯性轴承，的确，我真不擅长起名。
+
+# Payloads
+block.betamindy-payload-cannon.name = 载荷炮
+block.betamindy-payload-cannon.description =可以发射载荷。载荷的生命值越高，能够发射的距离越远，受到的伤害也越大，因此，造成的伤害也会越大。在[#98ffa9]inner range[]发射的载荷可以无伤害地发射。
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: inner range -> 安全范围内
Add space after =

File: assets/bundles/bundle_zh_CN.properties [hunk 16]
```diff
@@ -0,0 +1,747 @@
...
+block.betamindy-anucoin-node.description = 管理猫猫币和经济的中心责任建筑。能连接到产出或消费猫猫币的建筑。连接到猫猫仓库来提升上限。
+block.betamindy-anucoin-safe.name = \uE005猫猫保险箱
+block.betamindy-anucoin-safe.description = 增加连接银行的猫猫币的上限，在断开连接后猫猫币会回到银行。
+block.betamindy-anucoin-vault.name = \uE005猫猫仓库
+block.betamindy-anucoin-vault.description = 更大的猫猫保险箱，增加连接银行的猫猫币的上限，在断开连接后猫猫币会回到银行。
+
+block.betamindy-cafe.name = 小餐馆
+block.betamindy-cafe.description = 购买饮料和小吃，用猫猫币购买物品\n[accent]用贸易驿站获取猫猫币，并用猫猫银行分配。[]
+block.betamindy-ancient-store.name = 古董商店
+block.betamindy-ancient-store.description = 用猫猫币购买经典古老、再度流行的传说方块。商品每天更新！有些物品需要字节元素，与 [#ef8aff]Portal Invasion[]相关的物品在这里也可买到。
```
Comment (Weathercold) [inline assets/bundles/bundle_zh_CN.properties]: Portal Invasion -> 传送门入侵

**Review by Weathercold — CHANGES_REQUESTED** [review]

File: assets/bundles/bundle_zh_CN.properties (hunk 5 above)
Comment (woodjecket) [reply assets/bundles/bundle_zh_CN.properties:228]: 看一下原文，这里得和原版修复塔区别开

**Comment by woodjecket** [comment]
All above done

**Comment by Weathercold** [comment]
还有一些错误没有修改
https://github.com/sk7725/BetaMindy/pull/158#discussion_r925200340, https://github.com/sk7725/BetaMindy/pull/158#discussion_r925197426, https://github.com/sk7725/BetaMindy/pull/158#discussion_r925197251, https://github.com/sk7725/BetaMindy/pull/158#discussion_r925196951
空格都需要删掉，否则会在游戏内显示，例如`圣诞节 快乐 暨 新 年 快乐!`

**Comment by woodjecket** [comment]
@sk7725 please merge

**Pull request merged**